pkg/apis/metrics/exporter.go

```diff
@@ -295,7 +295,7 @@ func (e *Exporter) DiscoveryWatchedCHIs(kubeClient kube.Interface, chopClient *c
 			continue
 		}
 
-		if !chi.GetStatus().HasNormalizedCHICompleted() {
```

Comment:
`GetStatus` can return nil, so we could either:
* `if chi.GetStatus() == nil || !chi.GetStatus().HasNormalizedCHICompleted()`
* `if chi.EnsureStatus().HasNormalizedCHICompleted()`

I picked the second option for brevity (and because I'm thinking there's no harm to just ensuring that the status struct is non-nil on access like this, i.e., because it is synchronized and can't "clobber" any other concurrent mutations to this state).